library/public/image.h

```diff
@@ -168,6 +168,16 @@ class F3D_EXPORT image
    */
   std::vector<unsigned char> saveBuffer(SaveFormat format = SaveFormat::PNG) const;
 
+  /**
+   * Convert to colored text using ANSI escape sequences for printing in a terminal.
+   * Assumes terminal support for:
+   * - unicode block characters (U+2580, U+2584, U+2588)
```

Comment:
Please add a comment explaining that two pixel lines are written per terminal line

Reply:
And also something about the alpha cutoff.